kythe/javatests/com/google/devtools/kythe/analyzers/java/testdata/pkg/Deprecation.java

```diff
@@ -0,0 +1,25 @@
+package pkg;
+
+// - @+9Deprecation defines/binding DeprecationClass
```

Comment:
The verifier looks for exactly `//-`, so none of these assertions are being checked.

Reply:
Thanks, I think this got clobbered by the formatter. (The style guide technically requires a space after `//` in line comments, so it inserts them.)